Automattic/jetpack
# Search: Add animation to modal summoning

#### Changes proposed in this Pull Request:
* Slides in the search modal from bottom of the screen upon summoning.

![Large GIF (1788x808)](https://user-images.githubusercontent.com/4044428/112202096-a0162480-8bd6-11eb-8822-12574cb76ca1.gif)

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Apply this change to your site with a Jetpack Search subscription.
* Navigate to `/` and enter a query into a search input. Verify that the search modal appears with an entry animation.
* Navigate to `/?s=some-search-value`. Verify that the search modal appears with an entry animation.

Note that we currently have special handling when dismissing the search modal if the initial href includes `/?s=`. Dismissing the modal in this instance will strip all search-related query string values and navigate the window to the resulting href (e.g. `/`).
